kong/conf_loader/init.lua

```diff
@@ -486,13 +489,7 @@ local CONF_PARSERS = {
 
   lua_ssl_trusted_certificate = { typ = "array" },
   lua_ssl_verify_depth = { typ = "number" },
-  lua_ssl_protocols = {
-    typ = "string",
-    directives = {
-      "nginx_http_lua_ssl_protocols",
-      "nginx_stream_lua_ssl_protocols",
-    },
-  },
+  lua_ssl_protocols = { typ = "string" },
```

Comment:
We put these directives into a separate template file so that they can be reused when doing cmd injection. Looks like a better solution is keeping these directives in this table but adding them to the ingore list so that they won't be injected twice in nginx configuration and `conf.nginx_http_lua_ssl_protocols`, `conf.nginx_stream_lua_ssl_protocols` will be set.